docs/server/source/production-deployment-template/troubleshoot.rst

```diff
@@ -104,3 +104,35 @@ MongoDB instance correctly.
 
 More information about this configuration is provided in
 :doc:`this document <cloud-manager>`.
+
+6. Create a Persistent Volume from existing Azure disk storage Resource
+---------------------------------------------------------------------------
+While disintegrating k8s cluster dynamically created PVs and the underlying Azure storage 
+resources are also deleted thus cannot be used in a new cluster. This workflow will preserve
```

Comment:
This sentence is hard to read. Maybe:

When deleting a k8s cluster, all dynamically-created PVs are deleted,
along with the underlying Azure storage disks (so those can't be used in a new cluster).